modules/standard/BitOps.chpl

```diff
@@ -157,7 +157,7 @@ module BitOps {
  * module to hide the extern procedures
  */
 pragma "no doc"
```

Comment:
Yeah.  I just didn't because I made this patch a day ago and didn't get around to testing it until today and I figured it wasn't an interesting change.  Do you want me to do it this merge?